turtlebot3_gazebo/CMakeLists.txt

```diff
@@ -33,4 +33,4 @@ install(DIRECTORY models rviz worlds launch
 ament_export_dependencies(rclcpp)
 ament_export_dependencies(gazebo_ros_pkgs)
 ament_export_dependencies(turtlebot3)
```

Comment:
This line should be updated (or removed as suggested in comment above)

Reply:
My mistake. I've deleted.